articles/what-api-endpoints-to-expose-to-the-public-internet.md

```diff
@@ -61,12 +61,13 @@ If you would like to use Fleet's Windows MDM features, the following endpoints n
 
 If you would like to use Fleet's iOS/iPadOS MDM features, the following endpoints need to be exposed:
 
-- `/enroll`: Allows end users to access the enrollment page on which they download an enrollment profile to enroll their iOS/iPadOS host.
+- `/api/*/fleet/enroll`: Allows end users to access the enrollment page on which they download an enrollment profile to enroll their iOS/iPadOS host.
 - `/api/*/fleet/enrollment_profiles/ota`:  Allows hosts to download an enrollment profile.
 
 ### Android
 
-- `/enroll`: Allows end users to access the enrollment page where they select a link to enroll their Android host.
+- `/api/*/fleet/enroll`: Allows end users to access the enrollment page where they select a link to enroll their Android host.
```

Comment:
@BCTBB this endpoint should have the `/api/*/fleet/enroll` prefix to be consistent with other endpoints in this guide right?

This tweak is unrelated to the main addition on line 70.